Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XLGPlusPlus/DominoInvocationAnalyzer.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using BuildXL.Analyzers.Core.XLGPlusPlus;
+using BuildXL.Scheduler.Tracing;
+using BuildXL.ToolSupport;
+using BuildXL.Utilities;
+using Google.Protobuf;
+
+namespace BuildXL.Execution.Analyzer
+{
+
+    internal partial class Args
+    {
+        public Analyzer InitializeDominoInvocationAnalyzer()
+        {
+            string inputDirPath = null;
+            string outputFilePath = null;
+
+            foreach (var opt in AnalyzerOptions)
+            {
+                if (opt.Name.Equals("outputFile", StringComparison.OrdinalIgnoreCase) ||
+                   opt.Name.Equals("o", StringComparison.OrdinalIgnoreCase))
+                {
+                    outputFilePath = ParseSingletonPathOption(opt, outputFilePath);
+                }
+                else if (opt.Name.Equals("inputDir", StringComparison.OrdinalIgnoreCase) ||
+                     opt.Name.Equals("i", StringComparison.OrdinalIgnoreCase))
+                {
+                    inputDirPath = ParseSingletonPathOption(opt, inputDirPath);
+                }
+                else
+                {
+                    throw Error("Unknown option for event stats analysis: {0}", opt.Name);
+                }
+            }
+
+            return new DominoInvocationAnalyzer(GetAnalysisInput())

```

Comment:
It seems that InputDirPath is critical, perhaps a check to ensure the inputpath is not null would help users.